tests/boot/grub_test_snapshot.pm

```diff
@@ -13,13 +13,16 @@ use base 'opensusebasetest';
 use testapi;
 use power_action_utils 'power_action';
 use utils qw(workaround_type_encrypted_passphrase reconnect_mgmt_console);
-use bootloader_setup qw(stop_grub_timeout boot_into_snapshot);
+use bootloader_setup qw(stop_grub_timeout boot_into_snapshot change_grub_config);
 use Utils::Backends 'is_pvm';
+use Utils::Architectures qw(is_aarch64);
 
 sub run {
     my $self = shift;
 
     select_console 'root-console';
+    # disable GRUB_TIMEOUT to avoid this module failed since grub menu timeout
+    change_grub_config('=.*', '=-1', 'GRUB_TIMEOUT', '', 1) if (is_aarch64);
```

Comment:
There is no need to add the fourth argument `''` since it is already set to this in the [change_grub_config definition](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/bootloader_setup.pm#L1224). Still it doesn't hurt either.